src/docs/product/integrations/aws-lambda/index.mdx

```diff
@@ -94,22 +94,53 @@ Learn more about [how this integration works](/product/integrations/aws-lambda/h
 
 ## Serverless Framework
 
-Note that this integration will configure your existing lambda functions correctly to use the SDK layer and set the appropriate environment variables as per [how this integration works](/product/integrations/aws-lambda/how-it-works/).  You can check the output `aws lambda get-function-configuration --function-name <yourFunctioName>` and this works.
+Note that this integration will configure your existing lambda functions correctly to use the SDK layer and set the appropriate environment variables as per [how this integration works](/product/integrations/aws-lambda/how-it-works/).  You can check the output `aws lambda get-function-configuration --function-name <yourFunctionName>` and this works.
 However when you perform a subsequent `sls deploy` the environment variables are not maintained but it appears the reference to the layer is retained.  A workaround exists where you can set the layer definition and environment variables in `serverless.yml`.  There are two problems with this, you have to manually add the reference to the SDK layer for every function you want "Sentrified" and you can't maintain the latest layer version automatically.
+
+### Node
 ```
+provider:
+
+  # other configuration
+  region: <AWS_REGION>
+
   environment:
     SENTRY_TRACES_SAMPLE_RATE: '1.0'
-    SENTRY_DSN: '<your sentry dsn>'
+    SENTRY_DSN: '<SENTRY_DSN>'
     NODE_OPTIONS: '-r @sentry/serverless/dist/awslambda-auto'
 
-custom:
+layersResource:
   layers:
-    - arn:aws:lambda:${self:provider.region}:943013980633:layer:SentryNodeServerlessSDK:8
+    - arn:aws:lambda:${self:provider.region}:943013980633:layer:SentryNodeServerlessSDK:26
 #    - arn:aws:lambda:${self:provider.region}:943013980633:layer:SentryNodeServerlessSDK:latest
 
-myHandler:
-  handler: src/handlers/myHandler.handler
-  layers: ${self:custom.layers}
+<yourFunctionName>:
+  handler: <yourLambdaHandlerFunctionPath>
+  layers: ${self:layersResource.layers}
+```
+
+### Python
 ```
+provider:
+
+  # other configuration
+  region: <AWS_REGION>
+
+  environment:
+    SENTRY_TRACES_SAMPLE_RATE: '1.0'
+    SENTRY_DSN: '<SENTRY_DSN>'
+
+layersResource:
+  layers:
+    - arn:aws:lambda:${self:provider.region}:943013980633:layer:SentryPythonServerlessSDK:6
+#    - arn:aws:lambda:${self:provider.region}:943013980633:layer:SentryPythonServerlessSDK:latest
+
+<yourFunctionName>:
```

Comment:
Should we stick the function declaration under a [`functions:`](https://www.serverless.com/framework/docs/providers/aws/guide/functions/) or indent it one level?

Reply:
Nah we should stick with a declaration under `functions:` .. Updated it the snippet :)